modules/test_react_instrument/jsx/InputDate.js

```diff
@@ -0,0 +1,39 @@
+import InputBase from "./InputBase";
+
```

Comment:
I feel like you are trying to rewrite all the form elements in React, when such thing was already done [here](https://github.com/aces/Loris/blob/17.1-dev/jsx/Form.js)

Moreover, `<FormElement/>` accepts PHP Loris form representation as `formElements` prop. which usually is referred as`$this->form->form` in PHP code.